lib/rdoc/code_object/any_method.rb

```diff
@@ -110,7 +110,7 @@ def call_seq
   # See also #param_seq
 
   def call_seq= call_seq
-    return if call_seq.empty?
+    return if call_seq.nil? || call_seq.empty?
 
     @call_seq = call_seq
```

Comment:
It's not related to this PR but we may want to set `@call_seq` even when `call_seq` is `nil` or `""`:

```ruby
call_seq = call_seq if call_seq&.empty?
@call_seq = call_seq
```

Reply:
I thought about it but it's potentially risky if RDoc relies on this to avoid erasing the current value. Will need to do some testing to make sure it's ok to drop it.

Reply:
It makes sense.